OUTTAKES.md

```diff
@@ -30,3 +32,28 @@ we will see if the similarity measures we develop show that.
 | Bram Stoker                 | dracula.txt                     |
 | H. G. Wells                 | time_machine.txt                |
 | Edith Wharton               | ethan_frome.txt                 |
+
+---
+
+# Editor Recommendations
+
+However, because of this trait, it may 
```

Comment:
What does "this trait" refer to? And is this in reference to Python or are you including RStudio? At present, I don't think there is anything as powerful as RStudio for R, but I don't see that mentioned here.

Reply:
I moved this to `OUTTAKES.md` because it needs to go somewhere, but I'm not sure where - a chapter or appendix on "other tools"?